buildSrc/subprojects/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -244,6 +245,44 @@ class PerformanceTestPlugin : Plugin<Project> {
             channel = "adhoc"
             outputs.doNotCacheIf("Is adhoc performance test") { true }
         }
+
+        val shouldLoadScenariosFromFile = project.providers.gradleProperty("includePerformanceTestScenarios")
+            .forUseAtConfigurationTime()
+            .getOrElse("false") == "true"
+
+        configureSampleGenerators {
```

Comment:
So "sample" and "test project" is the same, right? Can we standardize on one or the other?

Reply:
Changed the method to `configureSampleGenerators()`.